configs/animations-ave.json

```diff
@@ -41,5 +42,48 @@
     "repo": "web-animations/web-animations-next",
     "shields": ["travis", "github/issues-pr", "github/issues"]
   }
+},
+"update-legend": {
+  "title": "Update Frequency Guidelines",
+  "lists": [
```

Comment:
Changed to shorter syntax, this results in some mangly looking JS code to get it to work with Polymer templates though.